src/Sidekick.UI/StartupExtensions.cs

```diff
@@ -12,6 +13,7 @@ public static IServiceCollection AddSidekickUIServices(this IServiceCollection s
         {
             services.AddSingleton<IViewLocator, ViewLocator>();
 
+            services.AddScoped<IApplicationLogViewModel, ApplicationLogViewModel>();
```

Comment:
@leMicin why scoped? Does WPF create scopes?

Reply:
We create scope through the use of the ViewLocator. For Sidekick, a scope is created for every opened window. This could allow us to have different windows opened of the same type with different data in them.

Reply:
We have a lot of services currently that should not be singleton. They were singleton for the only reason that we had a ton of legacy code and implementing DI with singleton is the easy first step.